RELEASES.md

```diff
@@ -1,3 +1,81 @@
+Version 1.21.0 (2017-10-12)
+==========================
+
+Language
+--------
+- [Relaxed path syntax. You can now add type parameters to values][43540]
+  Example:
+  ```
+  my_macro!(Vec<i32>::new); // Always worked
+  my_macro!(Vec::<i32>::new); // Now works
+  ```
+
+Compiler
+--------
+- [Upgraded jemalloc to 4.5.0][43911]
+- [Enabled unwinding panics on Redox][43917]
+- [Now emits a warning when you have an unused `extern crate`][42588]
+
+Libraries
+---------
+- [Generate builtin impls for `Clone` for `[T; N]` where `N` is between 0
+  and 32][43690]
```

Comment:
It's more than this. `Clone` is now implemented for all arrays with `T: Clone`, all tuples with every member being `Clone` and all function pointers.